articles/active-directory/develop/quickstarts/active-directory-ios.md

```diff
@@ -0,0 +1,180 @@
+> > [!div renderon="portal" id="makechanges" class="nextstepaction"]
+> > [Make this change for me]()
+>
+> > [!div id="appconfigured" class="hidden"]
+> > ![Already configured](../../../../includes/media/active-directory-develop-common/green-check.png) Your application is configured with this attribute
+
+## Step 2: Download your web server or project
+
+- [Download the XCode Project](https://github.com/Azure-Samples/active-directory-ios-swift-native-v2/archive/master.zip)
+
+## Step 3: Configure your project
+
+1. Extract the zip file and open the project in XCode.
+1. Edit **ViewController.swift** and replace the line starting with 'let kClientID' with the following code snippet:
+
+    ```csharp
+    private static string ClientId = "Enter_the_Application_Id_here";
+    ```
+1. Press Control + click **Info.plist** to bring up the contextual menu, and then select **Open As** > **Source Code**.
+1. Under the dict root node, add the following code:
+
+    ```xml
+    <key>CFBundleURLTypes</key>
+    <array>
+        <dict>
+            <key>CFBundleTypeRole</key>
+            <string>Editor</string>
+            <key>CFBundleURLName</key>
+            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+            <key>CFBundleURLSchemes</key>
+            <array>
+                <string>msal[Your_Application_Id_Here]</string>
+                <string>auth</string>
+            </array>
+        </dict>
+    </array>
+    ```
+
+## More Information
+
+Read these sections to learn more about this quickstart.
+
+### MSAL
+
+MSAL ([MSAL.framework](https://github.com/AzureAD/microsoft-authentication-library-for-objc)) is the library used to sign in users and request tokens used to access an API protected by Microsoft Azure Active Directory. You can use carthage to install it by first running the following command in bash terminal from App’s root folder to build MSAL:
+
+```bash
+echo "github \"AzureAD/microsoft-authentication-library-for-objc\" \"master\"" > Cartfile
+carthage update
+```
+
+Then, in XCode, open the **General**, go to **Linked Frameworks and Libraries** and add **MSAL.framework** in **Add other...**, then add the following content in **New Run Script Phase**:
+
+```text
+/usr/local/bin/carthage copy-frameworks
+```
+
+And the following to **Input Files**:
+
+```text
+$(SRCROOT)/Carthage/Build/iOS/MSAL.framework
+```
+
+### MSAL initialization
+
+You can add the reference for MSAL by adding the following code:
+
+```swift
+import MSAL
+```
+
+Then, initialize MSAL using the following code:
+
+```swift
+self.applicationContext = try MSALPublicClientApplication.init(clientId: kClientID, authority: kAuthority)
+```
+
+> |Where: ||
+> |---------|---------|
+> | `clientId` | The Application ID from the application registered in *portal.microsoft.com* |
+> | `authority` | The Azure AD v2.0 endpoint. In most of cases this will be *https<span/>://login.microsoftonline.com/common/v2.0* |
```

Comment:
`https://login.microsoftonline.com/common` is enough, please remove v2.0 from the end.